pkg/network/tracer/connection/ebpf_tracer.go

```diff
@@ -0,0 +1,733 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux_bpf
+
+package connection
```

Comment:
Yes, it is a copy of `connection/tracer.go`; nothing has changed from the old `connection/tracer.go` except for some variable renames.

I don't think there is a way to do this without messing up the git history unfortunately.

Reply:
Can't  you leave this in `connection/tracer.go` and avoid renaming the struct to `ebpfTracer` -- at least in the same commit ?

Reply:
Would that help with git blame? The change makes sense otherwise right? The large diff would still be present even if I split it out.

Reply:
I'm fairly certain it would, by leaving the file name the same git will detect that you're only changing the struct name. The way it shows now is that you wrote all of ebpf_tracer.go from scratch

Reply:
We would have to split `ebpfTracer` out into its own file eventually anyway, so that would break git blame in the same way in the future. `connection.Tracer` now had only one implementation before, so it made sense to have `tracer` in the same file. But I don't think it makes sense anymore to have it in the same file. IMHO it is fine to loose git blame over this re-org.

Reply:
I don't agree, but I will accept your decision to get this PR merged faster.

We can have a good file hierarchy and git blame by:
* make the new `tracer.go` called `tracer_api.go` 
* keeping this in tracer.go and renaming the class to ebpfTracer
* [optionally] opening a separate PR to do pure file renames which git can make sense of